src/components/AddResourceModal/ShareSocials.jsx

```diff
@@ -6,58 +6,75 @@ import {
   faInstagram,
   faGithub
 } from '@fortawesome/free-brands-svg-icons';
+import useIsMobile from 'hooks/useIsMobile';
+import PhillySkyline from 'icons/PhillySkyline';
 import styles from './AddResourceModal.module.scss';
 
-const ShareSocials = () => (
-  <div className={styles.modalContent}>
-    <Modal.Header className={styles.modalHeader} closeButton />
-    <p className={styles.socialShareMessage}>
-      You have successfully submitted a resource.
-    </p>
-    <h1 className={styles.greyHeader}>Thanks for sharing‼</h1>
-    <p className={styles.socialShareMessage} style={{ marginTop: 128 }}>
-      Follow us on social media:
-    </p>
-    <div className={styles.socialShareIconWrapper}>
-      <a
-        aria-label="Visit the PHLask community on Facebook"
-        className={styles.socialShareIcon}
-        href="https://www.facebook.com/pg/PHLASKecosystem/community/"
-        target="_blank"
-        rel="noopener noreferrer"
-      >
-        <FontAwesomeIcon icon={faFacebook} />
-      </a>
-      <a
-        aria-label="Follow PHLask on Twitter"
-        className={styles.socialShareIcon}
-        href="https://twitter.com/PHLASKecosystem"
-        target="_blank"
-        rel="noopener noreferrer"
-      >
-        <FontAwesomeIcon icon={faTwitter} />
-      </a>
-      <a
-        aria-label="Visit PHLask on Instagram"
-        className={styles.socialShareIcon}
-        href="https://www.instagram.com/phlaskecosystem/?hl=en"
-        target="_blank"
-        rel="noopener noreferrer"
-      >
-        <FontAwesomeIcon icon={faInstagram} />
-      </a>
-      <a
-        aria-label="View the source code on GitHub"
-        className={styles.socialShareIcon}
-        href="https://github.com/phlask/phlask-map"
-        target="_blank"
-        rel="noopener noreferrer"
-      >
-        <FontAwesomeIcon icon={faGithub} />
-      </a>
+const ShareSocials = () => {
+  const isMobile = useIsMobile();
+
+  return isMobile ? (
+    // MOBILE VIEW
+    <div className={styles.modalContent}>
+      <Modal.Header className={styles.modalHeader} closeButton />
```

Comment:
Avoid using the React Bootstrap library

Reply:
Good call! After some digging, I cleaned up all remaining references to the React Bootstrap library and the library itself from our dependencies